.circleci/enable_api.exp

```diff
@@ -16,6 +16,7 @@ expect -exact "Build Command:"
 send -- "\r"
 expect -exact "Start Command:"
 send -- "\r"
+log_user 0;
```

Comment:
Can this be shared with both the tests? Or can we set up the default profile first and then use it in both tests?